app/extend_block.go

```diff
@@ -13,7 +12,7 @@ import (
 // version.
 func ExtendBlock(data coretypes.Data, appVersion uint64) (*rsmt2d.ExtendedDataSquare, error) {
 	// Construct the data square from the block's transactions
-	dataSquare, err := square.Construct(data.Txs.ToSliceOfBytes(), appVersion, appconsts.SquareSizeUpperBound(appVersion))
+	dataSquare, err := square.Construct(data.Txs.ToSliceOfBytes(), appVersion, 64)
```

Comment:
[blocking][question] why hard-code `64` instead of using the `appconsts.SquareSizeUpperBound(appVersion)`?

Reply:
shit nice catch!!

Reply:
fixed in https://github.com/celestiaorg/celestia-app/pull/2012/commits/34957ae2b00a4471a170e0b6c9004233c9398d5e